pages/api/checkout/[pid].ts

```diff
@@ -1,13 +1,16 @@
 import { NextApiRequest, NextApiResponse } from 'next';
-import { getSession, setCheckout } from '../../../lib/auth';
+import { bigcommerceClient, getSession, setCheckout } from '../../../lib/auth';
 import { getCheckoutBody } from '../../../lib/checkout';
 
 export default async function checkout(req: NextApiRequest, res: NextApiResponse) {
     const { query: { pid } } = req;
 
     try {
-        const { storeHash } = await getSession(req);
-        const checkoutBody = getCheckoutBody(String(pid), storeHash);
+        const { accessToken, storeHash } = await getSession(req);
+        const bigcommerce = bigcommerceClient(accessToken, storeHash, 'v2');
+        const { account_uuid: merchantUuid } = await bigcommerce.get(`/store`);
```

Comment:
🍹 what happens if this call fails?

Reply:
should be caught in the catch below - we definitely need better error handling though. We have a ticket for that.